src/Rules/Library/ControlShouldSupportTablePattern.cs

```diff
@@ -25,15 +25,12 @@ public override EvaluationCode Evaluate(IA11yElement e)
         {
             if (e == null) throw new ArgumentNullException(nameof(e));
 
-            if (Patterns.Table.Matches(e))
-                return EvaluationCode.Pass;
-
-            return StringProperties.Framework.Is(Framework.Edge).Matches(e) ? EvaluationCode.Warning : EvaluationCode.Error;
+                return Patterns.Table.Matches(e) ? EvaluationCode.Pass : EvaluationCode.Error;
```

Comment:
This row is indented with one extra tab
```suggestion
            return Patterns.Table.Matches(e) ? EvaluationCode.Pass : EvaluationCode.Error;
```